services/crates/crates-user-downloads.service.js

```diff
@@ -0,0 +1,32 @@
+import { renderDownloadsBadge } from '../downloads.js'
+import { pathParams } from '../index.js'
+import { BaseCratesUserService, description } from './crates-base.js'
+
+export default class CratesUserDownloads extends BaseCratesUserService {
+  static category = 'downloads'
+  static route = {
+    base: 'crates',
+    pattern: 'udt/:userId',
+  }
+
+  static openApi = {
+    '/crates/udt/{userId}': {
+      get: {
+        summary: 'Crates.io User Total Downloads',
+        description,
+        parameters: pathParams({
+          name: 'userId',
+          example: '3027',
+          description:
+            'The user ID can be found using https://crates.io/api/v1/users/{username}',
```

Comment:
This renders as a link in the docs, but following it doesn't take you anywhere useful. Can we put it in `` `backticks` `` so it renders a `<code>` block instead of a link

Reply:
Yea, it looks better with `code block`
Made changes at 2982c8fa102456ec5ce627850d55c0dc058d8e2c